.devcontainer/scripts/postCreateCommand.sh

```diff
@@ -64,7 +64,7 @@ fi
 echo "#######################################################"
 echo "### Init submodules                                 ###"
 echo "#######################################################"
-git config --global --add safe.directory /workspaces/*
+git config --global --add safe.directory "*"
```

Comment:
this fix needs the Git > 2.35 and thus is dependent on https://github.com/eclipse-velocitas/devcontainer-base-images/pull/29